lib/model/unittest/CCountingModelTest.cc

```diff
@@ -3,6 +3,7 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+#include "CModelTestFixtureBase.h"
```

Comment:
We usually put same directory headers after our library headers.  (This applies to all 5 files.)

It relates to this sentence from the style guide:

> It is best practice to list the ML include files in the build order of the libraries they relate to, as this helps to catch accidental circular dependencies.

It would be especially true for this header as it currently defines a whole load of types in the global namespace.  I am not sure that's ideal, which is the subject of a different comment.